### PR TITLE
Created portfolio

### PR DESCRIPTION
**Pull request by Rehan-Khan-Engineer**

### CODE REVIEW

**Comment by Abbas4tech** [comment]
Group Similar Properties on entire style sheet @Rehan-Khan-Engineer

**Review by Rehmatulla-Khan — APPROVED** [review]

**Comment by Abbas4tech** [comment]
> Group Similar Properties on entire style sheet @Rehan-Khan-Engineer 
????

**Review by Tanvir04khan — APPROVED** [review]

**Review by Abbas4tech — DISMISSED** [review]

**Review by syedabdullahali — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,1375 @@
...
+  grid-template-columns: repeat(3, 1fr);
+  gap: 2rem;
+}
+
+.nav_link {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-size: var(--small-font-size);
+  color: var(--title-color);
```
[the remark this answers is not in the document]
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css]: Grouped Similar Properties @Abbas4tech

**Review by shaikh-7abish — APPROVED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,1500 @@
...
+/* For small devices */
+
+@media screen and (max-width: 350px) {
+  .container {
+    margin-left: var(--mb-1);
+    margin-right: var(--mb-1);
+  }
+
+  .nav_menu {
+    padding: 2rem 0.25rem 4 rem;
```
Comment (Tauqeer-Ahmed-99) [inline tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css:1136]: Where padding of 4rem will be applied and why and how?@Rehan-Khan-Engineer

File: tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,1500 @@
...
+  }
+
+  .nav_list {
+    column-gap: 0;
+  }
+
+  .home_content {
+    grid-template-columns: 0.25fr 3 fr;
```
Comment (Tauqeer-Ahmed-99) [inline tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css:1144]: Is it working or not? @Rehan-Khan-Engineer

**Review by Tauqeer-Ahmed-99 — DISMISSED** [review]

File: tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css (hunk 3 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css]: Corrected @Tauqeer-Ahmed-99

File: tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css (hunk 2 above)
Comment (Rehan-Khan-Engineer) [reply tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css:1136]: Where padding of 4rem will be applied : Bottom

**Review by anask601 — APPROVED** [review]

**Comment by nehalskdev** [comment]
I guess your approvals are done and still changes are remaining to make.

**Pull request merged**